conf/docker-aio/readme.txt

```diff
@@ -0,0 +1,23 @@
+first pass docker all-in-one image, intended for running integration tests against.
+
+Could be potentially usable for normal development as well.
+
+
+Initial setup (aka - do once):
+- Do surgery on glassfish4 and solr4.6.0 following guides, place results in `conf/docker-aio/dv/deps` as `glassfish4dv.tgz` and `solr-4.6.0dv.tgz` respectively.
```

Comment:
@pameyer can we automate this "surgery"? It's already automated in Vagrant. When you run `vagrant up` Glassfish and Solr are configured and ready to be used by Dataverse.

Reply:
Potentially; but if so this should probably be unified into a separate script (so that there aren't three places that need to be changed in the future).

Reply:
A little more documentation on these manual steps would be nice. For example:

- Download or create `glassfish4dv.tgz` and `solr-4.6.0dv.tgz `
- `mkdir conf/docker-aio/dv/deps`
- `cp ~/Downloads/glassfish4dv.tgz conf/docker-aio/dv/deps`
- `cp ~/Downloads/solr-4.6.0dv.tgz conf/docker-aio/dv/deps`

Reply:
I wasn't sure how to delete the docker container I had left running. I ran `docker --help`, crossed my fingers, and ran `docker kill adb505218f64` which seems to have worked. We may want to document this for Docker noobs like me.